Pipfile

```diff
@@ -11,6 +11,9 @@ allow_prereleases = true
 
 [packages]
 colorama = "*"
+dataclasses = "*"
+importlib_metadata = "*"
```

Comment:
We should be able to use the `importlib` stdlib alternative in python 3.8

Reply:
Does everybody use python 3.8? If not, @vkantsev might need to help everyone lagging behind to upgrade.

Reply:
Ah I got it. Will upgrade currently using Python 3.6.9

Reply:
Ok, so @luchesar would solve his problem this way. I could possibly do it too.

However, in order to make our code usable by others, we will need to instruct everyone interested in it to install python 3.8. Isn't it more reasonable to just work with 3.6, which seems to be the default on the huge majority of platforms and packages?

To put it otherwise, what are the specific benefits that 3.8 gives us so that they are worth giving up the easily available 3.6?

Reply:
If we can use the same libraries as per above in 3.6 should be fine by me to just include them.
I did but not sure why is the build failing.

Reply:
@mapto has right, we ought to standardise on the lowest common version
@luchesar i'm checking the build

Reply:
build fixed in 0fe880d